test/node/uuid.test.ts

```diff
@@ -62,51 +51,23 @@ describe('UUID', () => {
     expect(uuid2.toString()).to.equal(LOWERCASE_DASH_SEPARATED_UUID_STRING);
   });
 
-  /**
-   * @ignore
-   */
   it('should correctly create UUID from UUID (copying existing buffer)', () => {
     const org = new UUID();
     const copy = new UUID(org);
     expect(org.id).to.not.equal(copy.id);
     expect(org.id).to.deep.equal(copy.id);
   });
 
-  /**
-   * @ignore
-   */
-  it('should throw if passed invalid 36-char uuid hex string', () => {
```

Comment:
instead of deleting this test - do we instead want to modify it (/add a new test) that demonstrates that we do not enforce the UUID type?